scrapy/extensions/feedexport.py

```diff
@@ -480,6 +480,8 @@ def _handle_store_error(self, f, logmsg, spider, slot_type):
 
     def _handle_store_success(self, f, logmsg, spider, slot_type):
         logger.info("Stored %s", logmsg, extra={"spider": spider})
+        feed_uri = logmsg.split(":")[-1].strip()
+        self.crawler.stats.set_value(f"feedexport/feed_url/{feed_uri.split('/')[-1]}", feed_uri)
```

Comment:
Please, use alternatives like https://docs.python.org/3/library/pathlib.html or https://docs.python.org/3/library/urllib.parse.html to operate with URIs paths